docs/run_test/platform.rst

```diff
@@ -190,6 +191,9 @@ deployment.
   public IP addresses.  False means to connect with the private IP addresses.
   If not provided, the connections will default to using the public IP
   addresses.
+* **generate_public_address**. True means to create a public IP address for the
```

Comment:
Please explain, when use_public_address is False, why the create_public_address == True is a valid scenario about the default outbound.